pkg/authorization/authorizer/authorizer.go

```diff
@@ -59,15 +59,21 @@ func (a *openshiftAuthorizer) Authorize(ctx kapi.Context, passedAttributes Autho
 	return false, denyReason, nil
 }
 
+// GetAllowedSubjects returns the subjects it knows can perform the action.
+// If we got an error, then the list of subjects may not be complete, but it does not contain any incorrect names.
+// This is done because policy rules are purely additive and policy determinations
+// can be made on the basis of those rules that are found.
 func (a *openshiftAuthorizer) GetAllowedSubjects(ctx kapi.Context, attributes AuthorizationAttributes) (util.StringSet, util.StringSet, error) {
```

Comment:
No, it should just return error.  But it should return an error that either satisfies Aggregate interface (sufficient for now) or we create a specific type of error that callers can check against.  Returning an aggregate and not changing the signature is fine.